CoffeeChat/Controller/GroupsViewController.swift

```diff
@@ -51,14 +52,23 @@ class GroupsViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .backgroundWhite
+        view.backgroundColor = .backgroundLightGreen
 
-        searchBar.searchBarStyle = .minimal
         searchBar.delegate = self
+        searchBar.backgroundColor = .backgroundWhite
+        searchBar.backgroundImage = UIImage()
+        if let textField = searchBar.value(forKey: "searchField") as? UITextField {
```

Comment:
This seems a little jank: if the build target is iOS 13 then there's `.searchTextField` but otherwise this might be the best option.